test-harness/specs/parameters-ac10.oas3.txt

```diff
@@ -0,0 +1,34 @@
+====test====
+When I send a request to an operation
+And the operation has a required path param
+And the request path does not contain the param
+Then I should get a 422 response back
+And the payload of the response should be in application/problem+json content-type
+====spec====
+openapi: 3.0.0
+paths:
+  '/path/{param}/foo':
+    parameters:
+      - name: param
+        in: path
+        required: true
+        style: simple
+        schema:
+          type: string
+    get:
+      responses:
+        '200':
+          content:
+            application/json:
+              schema:
+                type: object
+                properties: {}
+====server====
+mock -p 4010 ${document}
+====command====
+curl -i http://127.0.0.1:4010/path//foo
+====expect====
+HTTP/1.1 422 Unprocessable Entity
+content-type: application/problem+json
+
+{"type":"https://stoplight.io/prism/errors#UNPROCESSABLE_ENTITY","title":"Invalid request body payload","status":422,"detail":"Your request body is not valid and no HTTP validation response was found in the spec, so Prism is generating this error for you.","validation":[{"location":["path"],"severity":"Error","code":"required","message":"should have required property 'param'"}]}
```

Comment:
```suggestion
{"type":"https://stoplight.io/prism/errors#UNPROCESSABLE_ENTITY","title":"Invalid request body payload","status":422,"detail":"Your request is not valid and no HTTP validation response was found in the spec, so Prism is generating this error for you.","validation":[{"location":["path"],"severity":"Error","code":"required","message":"should have required property 'param'"}]}
```

Not sure if this is a change for this PR, but maybe? Surely saying `request body` here ain't right

Reply:
Looks like not right, I'll check where is that coming from. Good spot!